src/OrchardCore/OrchardCore.Localization.Core/OrchardCore.Localization.Core.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Infrastructure.Abstractions\OrchardCore.Infrastructure.Abstractions.csproj" />
```

Comment:
Actually this is not the module. This file is the localization package, but I can see another reference to `Orchard.Abstractions` already which is wrong too.

Reply:
If I'm not wrong I added this to access `PathExtensions`

Reply:
Then it needs to be removed. Before we publish on Nuget.

Reply:
I will double check this one right now

Reply:
I tried a lot when I remove this reference the `PathExtensions` will be missing, when I look to other modules, they are using `OrchardCore.Module.Targets` which make it works fine

Reply:
@sebastienros what's the action needed here after my clarification?